common/src/test/java/org/apache/servicecomb/toolkit/common/TextCompareTest.java

```diff
@@ -43,7 +43,7 @@
   Path destPath = Paths.get("./src/test/resources/compare/HelloEndPoint2.yaml");
 
   @Test
-  public void contractCompareText() throws IOException {
+  public void testContractCompareText() throws IOException {
```

Comment:
We don't need to add test prefix for all the test methods which has the Test annotation.

Reply:
unified naming style,may be it can be keep